spec/views/content_items/filter.html.erb_spec.rb

```diff
@@ -2,24 +2,29 @@
 
 RSpec.describe "content_items/filter.html.erb", type: :view do
   describe "Form for filtering content items" do
-    it "will render an empty select box if supplied with no organisation" do
+    it "will render an empty select box if supplied with no data" do
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/95/commits/9d7671db4010f37bbbdcb1ddc66702140f365844